CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 1.3.1 (2019-10-07)
+
+- Fixed the list data provider demo to show all data provider functionality. [#23](https://github.com/blackbaud/skyux2-demos/pull/23)
```

Comment:
Should this be more specific? Something like...
```
...show examples of sorting, filtering, and searching while using the data provider.
```

Reply:
I went back and forth. I'm fine with being more specific @Blackbaud-AlexKingman. Updated

@blackbaud-johnly care to give it another review with the new wording?